--- volumes/wp/wp-load.php.ORIG	2019-03-14 08:30:45.000000000 +0100
+++ volumes/wp/wp-load.php	2019-03-14 08:52:13.000000000 +0100
@@ -31,6 +31,9 @@
  *
  * If neither set of conditions is true, initiate loading the setup process.
  */
+
+$wp_config_loaded = FALSE;
+
 if ( file_exists( ABSPATH . 'wp-config.php') ) {
 
 	/** The config file resides in ABSPATH */
@@ -41,6 +44,31 @@
 	/** The config file resides one level above ABSPATH but is not part of another installation */
 	require_once( dirname( ABSPATH ) . '/wp-config.php' );
 
+    $wp_config_loaded = TRUE;
+
 } else {
 
+    // Look for wp-config.php up the directory tree, up to a depth of
+    // 5 e.g. for wp-content/plugins/myplugin/inc/foo.php
+    for ( $depth = 1; $depth <= 5 ; $depth++ ) {
+
+        $parentdir = dirname( $_SERVER["SCRIPT_FILENAME"], $depth );
+        if ( @file_exists($wp_config = "$parentdir/wp-config.php" ) ) {
+
+            require_once( $wp_config );
+            $wp_config_loaded = TRUE;
+            break;
+
+        } else if ( FALSE !== @lstat( "$parentdir/wp-content" ) ) {
+
+            // We appear to have reached a WordPress top-level
+            // directory; stop here
+            break;
+
+        }
+    }
+}
+
+if ( ! $wp_config_loaded ) {
+
 	// A config file doesn't exist
